bonfire/config.py

```diff
@@ -54,17 +54,33 @@
 # set to true when bonfire is running via automation using a bot acct (not an end user)
 BONFIRE_BOT = os.getenv("BONFIRE_BOT")
 
-AUTO_ADDED_FRONTEND_DEPENDENCIES = (
+DEFAULT_FRONTEND_DEPENDENCIES = (
     "chrome-service",
     "frontend-configs",
     "rbac",
     "rbac-frontend",
     "host-inventory",
     "host-inventory-frontend",
-    "unleash-proxy"
+    "unleash-proxy",
 )
 
 
+def _get_auto_added_frontend_dependencies():
+    env_var = os.getenv("BONFIRE_FRONTEND_DEPENDENCIES")
+    result = set()
+
+    if env_var is None:
+        result = set(DEFAULT_FRONTEND_DEPENDENCIES)
+    elif env_var.strip() != "":
+        for app in env_var.split(","):
+            result.add(app.strip())
+
+    return result
```

Comment:
is there any situation where you don't expect this to be a string ? 

Also, for non-emptiness checks, I just realized we're adding empty elements here, regardless. So, something like:
```
BONFIRE_FRONTEND_DEPENDENCIES='  ,   '
```

will have an undesirable outcome :/ 

with a comprehension list maybe:

```suggestion
    env_var = os.getenv("BONFIRE_FRONTEND_DEPENDENCIES")
    
    if env_var is None:
        return set(DEFAULT_FRONTEND_DEPENDENCIES)
    
    return set([ x for x in env_var.split(",") if x.strip()])
```

Reply:
Good point, updated